app/api/business/brief/brief_edit_business.py

```diff
@@ -214,6 +215,18 @@ def edit_opportunity(user_id, brief_id, edits):
 
         errors.append(message)
 
+    if (
+        closing_date_was_edited(brief.closed_at.to_iso8601_string(), previous_data['closed_at']) and
+        not brief_blackout_validator.validate_closed_at_blackout(brief.closed_at.to_iso8601_string())
+    ):
+        blackoutDates = brief_blackout_validator.get_blackout_dates()
```

Comment:
It looks like you need to update this in multiple files. Casing should be underscore for Python.
```suggestion
        blackout_dates = brief_blackout_validator.get_blackout_dates()
```

Reply:
Sure, I'll change the variable name to follow Python conventions.